services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -75,52 +55,6 @@ class PlacementConstraintStr(ConstrainedStr):
     )
 
 
-class VFSCacheMode(str, Enum):
-    __slots__ = ()
-
-    OFF = "off"
-    MINIMAL = "minimal"
-    WRITES = "writes"
-    FULL = "full"
-
-
-class EnvoyLogLevel(StrAutoEnum):
-    TRACE = auto()
-    DEBUG = auto()
-    INFO = auto()
-    WARNING = auto()
-    ERROR = auto()
-    CRITICAL = auto()
-
-    def to_log_level(self) -> str:
-        lower_log_level: str = self.value.lower()
-        return lower_log_level
-
-
-class RCloneSettings(SettingsLibraryRCloneSettings):
-    R_CLONE_DIR_CACHE_TIME_SECONDS: PositiveInt = Field(
-        10,
-        description="time to cache directory entries for",
-    )
-    R_CLONE_POLL_INTERVAL_SECONDS: PositiveInt = Field(
-        9,
-        description="time to wait between polling for changes",
-    )
-    R_CLONE_VFS_CACHE_MODE: VFSCacheMode = Field(
-        VFSCacheMode.MINIMAL,
-        description="used primarly for easy testing without requiring requiring code changes",
-    )
-
-    @validator("R_CLONE_POLL_INTERVAL_SECONDS")
-    @classmethod
-    def enforce_r_clone_requirement(cls, v: int, values) -> PositiveInt:
-        dir_cache_time = values["R_CLONE_DIR_CACHE_TIME_SECONDS"]
-        if not v < dir_cache_time:
-            msg = f"R_CLONE_POLL_INTERVAL_SECONDS={v} must be lower than R_CLONE_DIR_CACHE_TIME_SECONDS={dir_cache_time}"
-            raise ValueError(msg)
-        return v
-
-
 class StorageSettings(BaseCustomSettings):
```

Comment:
don't we have that already somewhere else?

Reply:
We actually have 3 of them. Removed the one from director-v2